### PR TITLE
Updating description text for Create/Update virtual machines APIs documentation. No changes to specs.

### PR DESCRIPTION
**Pull request by avinashakka**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Can you be more specific by explicitly saying which settings cannot be changed?

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: I have added the statement about "a property value not being updatable" to some properties individually and some already exist. 

Since the same per OSProfile property description is shared for Put and Patch, adding this  "**Some of the settings cannot be changed once VM is provisioned.**" statement.

**Comment by yungezz** [comment]
looks good to me. once code owner comments resolved, ok to merge

**Comment by yungezz** [comment]
HI @hyonholee are all your comments address? is this PR good for you?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Pls. note some properties can be set only during virtual machine creation.

**Comment by yungezz** [comment]
hi @hyonholee is the PR  ok for you?

**Comment by yungezz** [comment]
HI @avinashakka what's the status of this PR? is it approved internally?

**Comment by yungezz** [comment]
ping again.

**Review by yungezz — APPROVED** [review]

**Comment by avinashakka** [comment]
> HI @avinashakka what's the status of this PR? is it approved internally?

Yes. I added the changes from review comment.

**Comment by yungezz** [comment]
> > HI @avinashakka what's the status of this PR? is it approved internally?
> 
> Yes. I added the changes from review comment.

is this PR ready to merge?

**Comment by avinashakka** [comment]
> > > HI @avinashakka what's the status of this PR? is it approved internally?
> > 
> > 
> > Yes. I added the changes from review comment.
> 
> is this PR ready to merge?

Yes. I have requested @smotwani to sign-off too.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: for your Linux VM

**Comment by yungezz** [comment]
> > > > HI @avinashakka what's the status of this PR? is it approved internally?
> > > 
> > > 
> > > Yes. I added the changes from review comment.
> > 
> > 
> > is this PR ready to merge?
> 
> Yes. I have requested @smotwani to sign-off too.

thanks. once @smotwani signed off, I'll merge the PR.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Pls. add a NOTE to the description, that do not put any secrets or passwords in the customData property.

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Done.

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Done.

**Review by hyonholee — APPROVED** [review]

**Comment by yungezz** [comment]
hi @avinashakka is this PR ready to merge?

**Comment by yungezz** [comment]
hi @NullMDR  could you pls help to look at SDK Net failure in this PR? thanks.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Pls. move the  statement "Do not pass any secrets or passwords in customData property." to a new line and prepend it with NOTE: and also see if we can highlight it.

Comment (avinashakka) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Done.

**Review by smotwani — APPROVED** [review]

**Comment by avinashakka** [comment]
> hi @avinashakka is this PR ready to merge?

Please merge the PR whenever you can. I have approval from all the required internal reviewers.

**Pull request merged**
